service/src/main/java/bio/terra/cbas/dependencies/sam/SamService.java

```diff
@@ -27,10 +44,42 @@ private StatusApi getStatusApi() {
     return new StatusApi(samClient.getApiClient());
   }
 
+  boolean hasPermission(String actionType) {
+    // don't check auth access with Sam if "sam.checkAuthAccess" is false
```

Comment:
OOC - what is the use case for having `sam.checkAuthAccess` to false? Is this a temporary placeholder so that we don't start checking these permissions until we're at a further stage of collaboration?

Reply:
`sam.checkAuthAccess` can be set to false when working locally so that you don't have to pass bearer token to requests.

Reply:
Ok, I will edit the RunSetsApiController to look at this configuration option before adding the user ID to RunSet, otherwise that will still error out locally.